user.go

```diff
@@ -29,19 +29,32 @@ type users struct {
 
 // User represents a Terraform Enterprise user.
 type User struct {
-	ID               string     `jsonapi:"primary,users"`
-	AvatarURL        string     `jsonapi:"attr,avatar-url"`
-	Email            string     `jsonapi:"attr,email"`
-	IsServiceAccount bool       `jsonapi:"attr,is-service-account"`
-	TwoFactor        *TwoFactor `jsonapi:"attr,two-factor"`
-	UnconfirmedEmail string     `jsonapi:"attr,unconfirmed-email"`
-	Username         string     `jsonapi:"attr,username"`
-	V2Only           bool       `jsonapi:"attr,v2-only"`
+	ID               string           `jsonapi:"primary,users"`
+	AvatarURL        string           `jsonapi:"attr,avatar-url"`
+	Email            string           `jsonapi:"attr,email"`
+	IsServiceAccount bool             `jsonapi:"attr,is-service-account"`
+	TwoFactor        *TwoFactor       `jsonapi:"attr,two-factor"`
+	UnconfirmedEmail string           `jsonapi:"attr,unconfirmed-email"`
+	Username         string           `jsonapi:"attr,username"`
+	V2Only           bool             `jsonapi:"attr,v2-only"`
+	IsSiteAdmin      bool             `jsonapi:"attr,is-site-admin"`
+	IsSsoLogin       bool             `jsonapi:"attr,is-sso-login"`
```

Comment:
These two attributes aren't always present-They are only returned for the current user (this struct is referenced by many other entities). For that reason, I think this should be a `*bool` type in order to allow nil values.

```suggestion
	IsSiteAdmin      *bool            `jsonapi:"attr,is-site-admin"`
	IsSsoLogin       *bool            `jsonapi:"attr,is-sso-login"`
```